scout/adapter/mongo/query.py

```diff
@@ -41,13 +41,16 @@ def build_case_query(
             case_query["status"] = {"$in": list(status)}
         return case_query
 
-    def delete_variants_query(self, case_id, variants_to_keep=[], min_rank_threshold=None) -> dict:
+    def delete_variants_query(
+        self, case_id, variants_to_keep=[], min_rank_threshold=None, exclude_ctg=[]
```

Comment:
This is nitpicking, but could it be perhaps keep_category - ctg makes me think config.. 😊

Reply:
Sure, better to be clear